docs/operation/operation.md

```diff
@@ -879,3 +879,55 @@ predicates and filters involved in the route processing:
   ]
 }
 ```
+
+## Profiling skipper
+
+Go profiling is explained in Go's
+[diagnostics](https://golang.org/doc/diagnostics.html) documentation.
+
+To enable profiling in skipper you have to use `-enable-profile`. This
+will start a profiling route at `/debug/pprof/profile` on the support
+listener, which defaults to `:9911`.
+
+### Profiling example
+
+Start skipper with enabled profiling:
+
+```
+./bin/skipper -routes-file examples/shadow-2-routes.eskip -enable-profile
```

Comment:
It would be nice to use an existing file (e.g from `./skptesting/`) so that users may actually try it out

Reply:
Before I thought to just use inline route, which does not depend on any CWD, but I missed to replace it :D